pnpm-lock.yaml

```diff
@@ -5850,7 +5853,7 @@ packages:
     dependencies:
       '@mdx-js/mdx': 2.2.1
       source-map: 0.7.3
-      webpack: 5.74.0
+      webpack: 5.74.0_@swc+core@1.2.203
```

Comment:
where are those changes from?

Reply:
No idea. I still need to polish this PR.

Reply:
Still no idea how it got here, but it should be fixed now.
Also moved it to devDeps.